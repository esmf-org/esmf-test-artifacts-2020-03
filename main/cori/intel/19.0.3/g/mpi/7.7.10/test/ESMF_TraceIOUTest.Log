build time -- 2021-10-26 20:37:42.760542
