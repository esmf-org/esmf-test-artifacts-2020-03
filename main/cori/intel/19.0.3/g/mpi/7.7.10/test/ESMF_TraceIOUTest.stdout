build time -- 2021-10-26 20:37:42.760542
/global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/preload.sh: line 3: 28918 Segmentation fault      env LD_PRELOAD="$LD_PRELOAD /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/libesmftrace_preload.so" $*
/global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/preload.sh: line 3: 28920 Segmentation fault      env LD_PRELOAD="$LD_PRELOAD /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/libesmftrace_preload.so" $*
/global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/preload.sh: line 3: 28917 Segmentation fault      env LD_PRELOAD="$LD_PRELOAD /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/libesmftrace_preload.so" $*
/global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/preload.sh: line 3: 28919 Segmentation fault      env LD_PRELOAD="$LD_PRELOAD /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_main/lib/libg/Unicos.intel.64.mpi.default/libesmftrace_preload.so" $*
srun: error: nid12990: tasks 0-3: Exited with exit code 139
srun: launch/slurm: _step_signal: Terminating StepId=48981389.36
